src/stackdriver.js

```diff
@@ -38,16 +53,16 @@ module.exports.toLogEntry = function (log, options = {}) {
   const data = { ...log, message }
   if (data.msg) { delete data.msg }
   if (data.labels) { delete data.labels }
-  if (data.httpRequest) { delete data.httpRequest }
 
   const entry = {
     meta: {
       resource: resource || { type: 'global' },
-      severity
+      severity,
+      trace: _getKey(log, data, 'trace', keys),
+      httpRequest: _getKey(log, data, 'httpRequest', keys)
```

Comment:
add these dynamically by looking at specified keys

Reply:
This is currently just set to `undefined` if key is not present, which should achieve the same behaviour - I've improved this and added additional tests in `2352d3d`  as well